sourcesapi/sourcesapi.py

```diff
@@ -0,0 +1,154 @@
+"""Handles collecting data from source talkers.
```

Comment:
I would implement this initially so that it's a setting to pick which source.
Trying to merge/decide at runtime what source to get will be complicated, and just providing the option of multiple sources is a huge feature.